crypto/arm_arch.h

```diff
@@ -182,4 +186,7 @@ extern unsigned int OPENSSL_armv8_rsa_neonized;
 
 # endif  /* defined __ASSEMBLER__ */
 
+# define IS_CPU_SUPPORT_UNROLL8_EOR3() \
+           (OPENSSL_armcap_P & ARMV8_UNROLL8_EOR3)
```

Comment:
is this more like a IS_CPU_PREFER_UNROLL8_EOR3 thing?
that means it is only a hint which code is expected to be faster, right?

Reply:
This means that both (1) the CPU supports the EOR3 instruction needed and (2) the CPU is "large" enough that it is expected that using the code that unrolls the loop by 8 and uses EOR will be faster

Reply:
ok.